feature-configs/demo/gatekeeper/sample-testcases.yaml

```diff
@@ -0,0 +1,49 @@
+---
+# This should fail to create due to the specific toleration
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: test-fail-specific-
+spec:
+  containers:
+  - image: docker.io/redhat/ubi8:latest
+    name: demo
+    command: ["/bin/sh", "-c", "while :; do date; sleep 10; done"]
```

Comment:
can't you just sleep as the entry point?

Reply:
I always do a while ; date ; sleep so the logs show constant updates.  Agree that just a sleep by itself would be functionally similar.

Reply:
Fine by me

Reply:
command: ["/bin/bash", "-c", "sleep INF"]